gasprice/gaspricesuggestor.go

```diff
@@ -0,0 +1,43 @@
+package gasprice
+
+import (
+	"context"
+	"time"
+
+	"github.com/0xPolygonHermez/zkevm-node/etherman"
+	"github.com/0xPolygonHermez/zkevm-node/log"
+	"github.com/0xPolygonHermez/zkevm-node/state"
+)
+
+// L2GasPricer interface for gas price suggestor.
+type L2GasPricer interface {
+	UpdateGasPriceAvg()
+}
+
+// NewL2GasPriceSuggestor init.
+func NewL2GasPriceSuggestor(ctx context.Context, cfg Config, pool pool, ethMan *etherman.Client, state *state.State) {
```

Comment:
~~Suggestor~~ -> **Suggester**

everywhere this word is found it should be fixed.

Reply:
From what I see on the dictionary both words are the same.
![image](https://user-images.githubusercontent.com/11256256/214069604-27b2f72a-2c5a-49ca-b83c-d894e6f9a518.png)

Reply:
Source?

you can check by yourself on Google: https://translate.google.com/?sl=en&tl=es&text=Suggestor%0A&op=translate

![image](https://user-images.githubusercontent.com/1298871/214072988-6d2af8aa-2c8b-4883-aaf8-8ed14ed831a5.png)

Reply:
Ok, I'll change it